<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GisAddMapNodeDialog</class>
 <widget class="QWidget" name="GisAddMapNodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,0">
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../qrc/images_default.qrc">:/images/default/map/addGIS_16.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string notr="true">TK_Title</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../qrc/images_default.qrc">
         <normaloff>:/images/default/servermanager/delete_16.png</normaloff>:/images/default/servermanager/delete_16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TK_MapName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TK_ParentNode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxMapTree">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonConfirm">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">PushButton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../qrc/images_default.qrc"/>
 </resources>
 <connections/>
</ui>
